### PR TITLE
EZBSystems,Inc.EasyBoot version 6.6.0.800

### PR DESCRIPTION
**Pull request by JoaquinPro64**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/53975)

### CODE REVIEW

File: manifests/e/EZBSystems,Inc/EasyBoot/6.6.0.800/EZBSystems,Inc.EasyBoot.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: EZBSystems,Inc.EasyBoot
```
Comment (ItzLevvie) [inline manifests/e/EZBSystems,Inc/EasyBoot/6.6.0.800/EZBSystems,Inc.EasyBoot.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: EZBSystems.EasyBoot
```

File: manifests/e/EZBSystems,Inc/EasyBoot/6.6.0.800/EZBSystems,Inc.EasyBoot.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: EZBSystems,Inc.EasyBoot
```
Comment (ItzLevvie) [inline manifests/e/EZBSystems,Inc/EasyBoot/6.6.0.800/EZBSystems,Inc.EasyBoot.installer.yaml:4]:
```suggestion
PackageIdentifier: EZBSystems.EasyBoot
```

File: manifests/e/EZBSystems,Inc/EasyBoot/6.6.0.800/EZBSystems,Inc.EasyBoot.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: EZBSystems,Inc.EasyBoot
```
Comment (ItzLevvie) [inline manifests/e/EZBSystems,Inc/EasyBoot/6.6.0.800/EZBSystems,Inc.EasyBoot.yaml:4]:
```suggestion
PackageIdentifier: EZBSystems.EasyBoot
```

File: manifests/e/EZBSystems,Inc/EasyBoot/6.6.0.800/EZBSystems,Inc.EasyBoot.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: EZBSystems,Inc.EasyBoot
+PackageVersion: 6.6.0.800
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://www.ezbsystems.com/dl1.php?file=ezb6_en.exe
+  InstallerSha256: 94B8C0103FF889D509F701E65058169D7B66409FC603A3571A9F7D4F9F148EA3
```
Comment (ItzLevvie) [inline manifests/e/EZBSystems,Inc/EasyBoot/6.6.0.800/EZBSystems,Inc.EasyBoot.installer.yaml:10]:
```suggestion
  InstallerSha256: 94B8C0103FF889D509F701E65058169D7B66409FC603A3571A9F7D4F9F148EA3
  ProductCode: EasyBoot_is1
```

File: manifests/e/EZBSystems,Inc/EasyBoot/6.6.0.800/EZBSystems,Inc.EasyBoot.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: EZBSystems,Inc.EasyBoot
+PackageVersion: 6.6.0.800
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://www.ezbsystems.com/dl1.php?file=ezb6_en.exe
```
Comment (ItzLevvie) [inline manifests/e/EZBSystems,Inc/EasyBoot/6.6.0.800/EZBSystems,Inc.EasyBoot.installer.yaml:9]:
```suggestion
  InstallerUrl: https://dw.ezbsystems.com/ezb6_en.exe
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/e/EZBSystems,Inc/EasyBoot/6.6.0.800/EZBSystems,Inc.EasyBoot.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: EZBSystems,Inc.EasyBoot
+PackageVersion: 6.6.0.800
+PackageLocale: en-US
+Publisher: EZB Systems, Inc.
+PackageName: EasyBoot
+License: (c) EZB Systems, Inc.
```
Comment (ItzLevvie) [inline manifests/e/EZBSystems,Inc/EasyBoot/6.6.0.800/EZBSystems,Inc.EasyBoot.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You will also need to update the publisher folder from `EZBSystems,Inc` to `EZBSystems`; as well as update file names in the YAML files from `EZBSystems,Inc.EasyBoot` to `EZBSystems.EasyBoot`

**Comment by JoaquinPro64** [comment]
> You will also need to update the publisher folder from `EZBSystems,Inc` to `EZBSystems`; as well as update file names in the YAML files from `EZBSystems,Inc.EasyBoot` to `EZBSystems.EasyBoot`

How do you change the publisher folder and YAML file name

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
> How do you change the publisher folder and YAML file name?

You can press the `.` (dot) on your keyboard on this PR which will bring up Visual Studio Online where you can make your changes.

**Comment by JoaquinPro64** [comment]
> > How do you change the publisher folder and YAML file name?
> 
> You can press the `.` (dot) on your keyboard on this PR which will bring up Visual Studio Online where you can make your changes.

Theres another folder with the same name already (i didnt submit that one) mine is EZBSystemsInc

Screenshot_23

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
> Theres another folder with the same name already (i didnt submit that one) mine is EZBSystemsInc

You can put EasyBoot in the `EZBSystems` folder so there aren't any duplicates.

**Comment by JoaquinPro64** [comment]
> > Theres another folder with the same name already (i didnt submit that one) mine is EZBSystemsInc
> 
> You can put EasyBoot in the `EZBSystems` folder so there aren't any duplicates.

ok, I changed it

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
> ok, I changed it

You need to commit the changes as well so the PR will be updated with the new changes.

**Comment by JoaquinPro64** [comment]
> > ok, I changed it
> 
> You need to commit the changes as well so the PR will be updated with the new changes.

Visual studio gives me an erro when trying to commit the changes: "Submitting review failed. Submitting review failed, no pending review for current pull request: 53965"

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
See https://github.com/microsoft/winget-pkgs/pull/53975#issuecomment-1066652170

**Comment by JoaquinPro64** [comment]
> > Visual studio gives me an error when trying to commit the changes: "Submitting review failed. Submitting review failed, no pending review for current pull request: 53965"
> 
> Interesting.
> 
> I made a PR on your fork to fix it which can be seen at [JoaquinPro64#1](https://github.com/JoaquinPro64/winget-pkgs/pull/1). All you have to do is merge that PR which will automatically update this PR.

hello, i just merged it, but i also have to merch this one https://github.com/microsoft/winget-pkgs/pull/53960 (IHMC - CmapTools) because it gives the same error

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**